scripts/cycling_upload_tiles.sh

```
@@ -0,0 +1,51 @@
+  exit 1
+fi
+
+count=$(find "$TILES_DIR" -name '*.bin' | wc -l | tr -d ' ')
+echo "uploading $count tiles from $TILES_DIR to r2://$BUCKET/$PREFIX (concurrency=$CONCURRENCY)"
+
+# xargs で並列に `wrangler r2 object put --remote` を回す。エラー時は
+# set -e と xargs の終了コードで全体 fail させる。
+find "$TILES_DIR" -name '*.bin' -print0 \
+  | xargs -0 -P "$CONCURRENCY" -I{} bash -c '
+      f="$1"
+      key="'"$PREFIX"'$(basename "$f")"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)